src/components/ForgotPassword/ForgotPasswordForm.css

```diff
@@ -0,0 +1,70 @@
+/**
+ * Styling for login and sign up
+ */
+@import "@folio/stripes-components/lib/variables.css";
```

Comment:
@anton-plotnikov this is pretty cool. I didn't realize we could import these variables from `stripes-components`

Reply:
seemed convenient to me